code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -42,13 +42,16 @@
 	maturation = 10
 	production = 1
 	yield = 1 //seeds if there isn't a dna inside
+	instability = 15 //allows it to gain reagent genes from nearby plants
 	potency = 30
 	var/volume = 5
+	reagents_add = list(/datum/reagent/water = 0.01) //ensures the resulting podperson will have water blood if no other genes are present
```

Comment:
```suggestion
```
remove this line and just make the blood type default to water if no valid chem production traits are present. it's possible to just remove this water production trait with botany tools, so you still have to have a way to handle a podperson growing from replica pod seeds with no chem production traits anyway.

Reply:
already have water as the default blood, that was added during a troubleshooting session and I just forgot to remove it